tests/test_helpers.py

```diff
@@ -35,7 +35,15 @@
     (["helper", "l_grants", "-c", "-d", "2020-05-25"],
      "sym2.0, awardnr: , acctn: , 2019-06-01 to 2030-12-31\n, awardnr: , acctn: GG012345, 2015-10-01 to 2025-09-30\n"
      ),
-
+     (["helper", "l_contacts", "-n", "ny"],
```

Comment:
do you have a test for the dates logic?

Reply:
Unfortunately not, because there's no date information for the contact in the sample database file. Should I modify that file and add in that information?